modules/protect/math-fallback.php

```diff
@@ -124,11 +124,19 @@ static function math_form() {
 			?>
 			<div style="margin: 5px 0 20px;">
 				<strong><?php esc_html_e( 'Prove your humanity:', 'jetpack' ); ?> </strong>
-				<?php echo $num1 ?> &nbsp; + &nbsp; <?php echo $num2 ?> &nbsp; = &nbsp;
-				<input type="input" name="jetpack_protect_num" value="" size="2" />
-				<input type="hidden" name="jetpack_protect_answer" value="<?php echo $ans; ?>" />
+				<label for="sum">
+					<?php esc_html( (string) $num1 ); ?>
```

Comment:
The `esc_html` doesn't print anything. You must use `echo` here. Also, you can leave the casting aside.
```php
<?php echo esc_html( $num1 ); ?>
```

Reply:
Noting that you can also use `<?php esc_html_e(  $num1 ); ?>` to output this directly without `echo`, if you prefer.

https://codex.wordpress.org/Function_Reference/esc_attr_e
https://codex.wordpress.org/Data_Validation#Text_Nodes

Reply:
Yes, I'm aware of `esc_html_e` and I already use it for 'plus' and 'equals' text string. I chose not to use it for `$num1` `$num2` `$ans` because they're all random numbers and retrieving the translation becomes redundant. The code responsible `$num1` `$num2` `$ans` is cut off in the code summary view.

Thanks for the help.